### PR TITLE
refactor: dsm headers and fix bugs

### PR DESCRIPTION
**Pull request by samim-hossain-mondal**
**snapshot for reference**
Screenshot 2023-03-31 at 3 22 31 PM

### CODE REVIEW

File: src/components/teamInformation/TeamInformations.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ function CardList() {
           console.log(response)
```
Comment (Abhishek-Kharpal) [inline src/components/teamInformation/TeamInformations.js:169]: Please remove this log

File: src/components/teamInformation/TeamInformations.js [hunk 2]
```diff
@@ -330,16 +330,16 @@ useEffect(()=>{
...
-                        spacing={2}
-                        rowGap={2}
-                        // justifyContent="center"
-                        alignItems="center"
-                
+                  className="body"
+                  container
+                  spacing={2}
+                  rowGap={2}
+                  // justifyContent="center"
```
Comment (Abhishek-Kharpal) [inline src/components/teamInformation/TeamInformations.js:337]: remove this comment if not needed

Comment (samim-hossain-mondal) [reply src/components/teamInformation/TeamInformations.js]: done

**Review by ritikrajdev — APPROVED** [review]
LGTM, Good Work !!

**Pull request merged**